### ISSUE TITLE
Add ability to link+open sections that are currently collapsed/hidden

### ISSUE THREAD

**Issue by JamoCA** [webhintio/webhint.io#865]

**Comment by molant** [comment]
Original Twitter conversation for more context:

https://twitter.com/gamesover/status/1197534627667357702

We could add an id to the summary and some JS to autoexpand on load

@sarvaje what do you think?

**Comment by sarvaje** [comment]
yep, we will need some js work there, because the documentation is autogenerated.
